elfi/methods/bo/acquisition.py

```diff
@@ -192,16 +192,16 @@ class LCBSC(AcquisitionBase):
 
     """
 
-    def __init__(self, *args, delta=None, **kwargs):
+    def __init__(self, *args, delta=None, include_prior=False, **kwargs):
         """Initialize LCBSC.
 
         Parameters
         ----------
-        delta : float, optional
+        delta: float, optional
             In between (0, 1). Default is 1/exploration_rate. If given, overrides the
             exploration_rate.
-        args
-        kwargs
+        include_prior: bool, optional
+            If false, then use a likelihood object, else use a posterior object.
```

Comment:
this is the description that i dislike a little bit. it is true that when this acquisition is used in BOLFIRE, `include_prior` converts log-likelihood-ratio optimisation into log-posterior optimisation, but that is not the case with all methods that use this same acquisition function. hence i would propose a comment like "if true, add negative log prior to model evaluations" (because whatever is modelled, this is what will be added to that) and possibly even "used in the BOLFIRE method" in case it may seem weird to someone that neg log prior is sometimes added to surrogate model evaluations at this point